README.md

```diff
@@ -29,11 +29,11 @@ Hive users have a choice of 3 runtimes when executing SQL queries.
 Users can choose between Apache Hadoop MapReduce, Apache Tez or
 Apache Spark frameworks as their execution backend. MapReduce is a
 mature framework that is proven at large scales. However, MapReduce
-is a purely batch framework, and queries using it may experience
+is a pure batch framework, and queries using it may experience
```

Comment:
Which grammar rule is getting violated,
https://www.vocabulary.com/dictionary/purely#:~:text=So%20they%20get%20purely%20confused,related%20letter%20is%20only%20business.

purely also seems correct to me

Reply:
Can adverbs modify nouns? I though it should be adjective.